doc/modules/mod_extdisco.md

```diff
@@ -0,0 +1,36 @@
+### Module Description
+Implements [XEP-0215: External Service Discovery](http://xmpp.org/extensions/xep-0215.html) for discovering information about services external to the XMPP network.
+The main use-case is to help discover STUN/TURN servers to allow for negotiating media exchanges.
+
+### Options
+* **type** (atom, default: unset) - service type
+
+    * **Valid values:** `stun, turn`
+
+#### Service Options
+* **host** (string, required, default: unset): Hostname or an IP address where the service is hosted.
+* **port** (string, required, default: unset): The communications port to be used at the host.
```

Comment:
The `port` option is recommended but not required AFAIK https://xmpp.org/extensions/xep-0215.html#table-1